src/features/zaaksysteem/ZaakZoeker.vue

```diff
@@ -1,56 +1,50 @@
 <template>
-  <section class="grid">
-    <section>
-      <form @submit.prevent="zoekOpZaak">
-        <fieldset class="utrecht-form-fieldset">
-          <label for="zaaknummer" class="utrecht-form-label">Zaaknummer</label>
-          <input
-            type="text"
-            id="zaaknummer"
-            v-model="store.zaakSearchParams.zaaknummer"
-            class="utrecht-textbox utrecht-textbox--html-input"
-            v-on:keydown.enter.prevent="zoekOpZaak"
-          />
-        </fieldset>
-        <menu>
-          <utrecht-button modelValue type="submit">Zoek</utrecht-button>
-        </menu>
-      </form>
-
-      <form @submit.prevent="zoekOpBsn">
-        <fieldset class="utrecht-form-fieldset">
-          <label for="bsn" class="utrecht-form-label">Bsn</label>
-          <input
-            type="text"
-            id="bsn"
-            v-model="store.zaakSearchParams.bsn"
-            class="utrecht-textbox utrecht-textbox--html-input"
-            placeholder="205827123"
-          />
-        </fieldset>
-        <menu>
-          <utrecht-button modelValue type="submit">Zoek</utrecht-button>
-        </menu>
-      </form>
-    </section>
-    <section>
-      <application-message
-        v-if="error"
-        messageType="error"
-        message="Er is een probleem opgetreden."
-      ></application-message>
+  <section>
+    <form class="search-bar" @submit.prevent="zoekOpZaak">
+      <label>
+        <input
+          type="search"
+          placeholder="Zoek op zaaknummer"
+          v-model="store.zaakSearchParams.zaaknummer"
+          title="0612345789 test@conduction.nl"
```

Comment:
let's make the title ZAAK-1, if we're going to keep using the title for testdata hints

Reply:
Yep, nice catch.